strawberryfields/future/apps/graph/sample.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+r"""
+Graph sampling
+==============
+
+**Module name:** :mod:`strawberryfields.future.apps.graph.sample`
+
+.. currentmodule:: strawberryfields.future.apps.graph.sample
+
+This module provides functionality for sampling of subgraphs from undirected graphs. The
+:func:`sample_subgraphs` function generates raw samples from
+:mod:`strawberryfields.future.apps.sample` and converts them to subgraphs using
+:func:`to_subgraphs`.
+
+Summary
+-------
+
+.. autosummary::
+    SAMPLE_DEFAULTS
+    sample_subgraphs
+    to_subgraphs
+
+Code details
+------------
+"""
+from typing import Optional
+import networkx as nx
+import numpy as np
+from .. import sample
```

Comment:
I generally prefer absolute imports vs. relative imports (mostly because they are more readable), and also pep8 recommends! https://www.python.org/dev/peps/pep-0008/#imports

I know this is a migration PR but might be worthwhile to change all the import paths to be absolute as part of this PR. In this case, this line could be rewritten as:
```
from strawberryfields.future.apps import sample
```

Reply:
Thanks @zzzeid, I'm on the fence for this one, I thought the relative imports would be better since things are buried relatively deep, and I saw pep8 is fine for explicit relative imports in this case. I can be persuaded though :grin:.

Reply:
This is generally more a matter of library convention than anything (and can depend on how the test suite is set up/the recommended way of installing or developing). The approach we've taken so far in Strawberry Fields and PennyLane has been:

* Relative imports for files in the same subpackage
* Absolute imports for files in different subpackages

This is mostly just for readability.